thirdparty/build_gstreamer.sh

```diff
@@ -0,0 +1,8 @@
+cd gst-build
+./gst-worktree.py add gst-build-1.16 origin/1.16
+cd gst-build-1.16
+git checkout 1.16.2
+sudo /root/.local/bin/meson --prefix=${PWD}/outInstall builddir -Dpython=disabled
+ninja -C builddir
+sudo /root/.local/bin/meson install -C builddir
+cd .. 
```

Comment:
we went 2 levels deep into cd so we should go up twice..
would cd ../.. will be better @joiskash